### DISCUSSION TITLE
Window to run a native page

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [zauberzeug/nicegui#3599] [Q&A] [answered]
### Question

I have a one-page demo written in kivy. Is it possible to run this demo in a nicegui window?

**Comment by python-and-novella** [comment]
Do you mean `iframe` in nicegui's page?

**Reply by dineshbvadhia** [reply]
If a python kivy program can run in an iframe from a nicegui page then yes ?

**Comment by python-and-novella** [comment, marked as answer]
nicegui supports `iframe`.
```python3
ui.element('iframe').props('src="https://nicegui.io" sandbox').classes('w-full h-[calc(100vh-3em)]')
```

**Comment by FruitfulApproach** [comment]
Iframe is nice, I'ma use it :)
